src/core/public/doc_links/doc_links_service.ts

```diff
@@ -270,6 +270,8 @@ export class DocLinksService {
         },
         transforms: {
           guide: `${ELASTICSEARCH_DOCS}transforms.html`,
+          // TODO add valid docs URL
+          alertingRules: `${ELASTIC_WEBSITE_URL}guide/en/machine-learning/${DOC_LINK_VERSION}/ml-configuring-alerts.html`,
```

Comment:
Are you planning to address this TODO is the current PR?

Reply:
There is no docs page for it yet, so I'm going to add it in the follow-up PR probably as soon as it's ready

Reply:
@darnautov This is the correct link.
```suggestion
          // TODO add valid docs URL
          alertingRules: `${ELASTICSEARCH_DOCS}transform-alerts.html`,
```